app/src/main/res/values/strings.xml

```diff
@@ -14,13 +14,44 @@
   <string name="menu_read_aloud">Read aloud</string>
   <string name="menu_read_aloud_stop">Stop reading aloud</string>
   <string name="menu_support_kiwix">Support Kiwix</string>
+  <string name="menu_host_books">Host Books</string>
   <string name="save_media">Save Media</string>
   <string name="save_media_error">An error occurred when trying to save the media!</string>
   <string name="save_media_saved">Saved media as %s to Android/media/org.kiwix…/</string>
   <string name="rescan_fs_warning">Scanning for downloadable content, please wait…</string>
   <string name="search_label">Search</string>
   <string name="choose_file">Select a Content File (*.zim)</string>
   <string name="open_in_new_tab">Open link in new tab?</string>
+  <string name="error_nozimfilesfound">No content found on your device.\nTake a look at the Help Page to get directions on how to load content into Kiwix.\nIf you did put a ZIM file on your device/external storage, you might retry in a minute or restart your device.</string>
```

Comment:
Delete this line if you're not using it. I have checked the develop branch and it is not being used anywhere.

Reply:
Fixed